privacyidea/static/app.js

```diff
@@ -76,6 +76,10 @@ myApp.constant("smtpServerUrl", backendUrl + instance + "/smtpserver");
 myApp.constant("radiusServerUrl", backendUrl + instance + "/radiusserver");
 myApp.constant("privacyideaServerUrl", backendUrl + instance + "/privacyideaserver");
 myApp.constant("recoveryUrl", backendUrl + instance + "/recover");
+myApp.constant("resourceNamePattern", "^[a-zA-Z0-9_-]+$");
+/* we have to ignore "test" and "test_request" as a resource name explicitly */
+myApp.constant("resourceNamePatternWoTest", "^(?!test$)([A-Za-z0-9_-]+)$");
+myApp.constant("resourceNamePatternWoTestRequest", "^[a-zA-Z0-9_-]+$");
```

Comment:
I think the "test_request" is missing here? It looks like the "resourceNamePattern".